bower.json

```diff
@@ -14,5 +14,17 @@
     "angular-ui-router": "~0.2.18",
     "bootstrap": "~3.3.6",
     "owasp-password-strength-test": "~1.3.0"
+  },
+  "overrides": {
+    "bootstrap": {
+      "main": [
+        "dist/css/bootstrap.css",
+        "dist/css/bootstrap-theme.css",
+        "less/bootstrap.less"
+      ]
+    }
+  },
+  "resolutions": {
+    "angular": "~1.3"
```

Comment:
We recently removed this configuration when upgrading to Angular 1.5. I'm not sure why it was removed, but I was having issues locally installing Bower with the package's postinstall script, because the version of Angular could not be resolved. My vote is to have this back added, but I think we should set it to `"~1.5"`.

@codydaig @ilanbiala @lirantal